docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+
+services:
+
+  postgres:
+    image: postgres:9.6
+    restart: always
+    ports:
+      - "5432:5432"
+    networks:
+      - hocs-network
+    environment:
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: dev
+    tmpfs:
+      - /var/lib/postgresql
+
+  localstack:
+    image: localstack/localstack:0.11.1
+    ports:
+      - "4566:4566"
+    networks:
+      - hocs-network
+    environment:
+      HOSTNAME_EXTERNAL: localstack
+      DEFAULT_REGION: eu-west-2
+      SERVICES: sqs
```

Comment:
We probably need the aws_cli stuff for creating the domain because since LocalStack 0.11.1 it's lazily loaded. https://github.com/UKHomeOffice/hocs/commit/14f00d8f92c1dbbec548823875f1e2d2512afa0d

Might be able to embed this within the app though..?